File: tmp/fix_sidebar.py
import sys

path = r"d:\Horntech\resources\views\admin\body\sidebar.blade.php"
with open(path, 'r', encoding='utf-8') as f:
    lines = f.readlines()

# Look for Company::first()
new_lines = []
skip = 0
for i, line in enumerate(lines):
    if skip > 0:
        skip -= 1
        continue
    if "App\Models\Company::first()" in line and i > 0 and "@php" in lines[i-1]:
        # We found it! Remove previous line (@php), current line, and next line (@endphp)
        new_lines.pop() # Remove @php
        skip = 1 # Skip @endphp
        continue
    new_lines.append(line)

with open(path, 'w', encoding='utf-8') as f:
    f.writelines(new_lines)

print("Done")

File: tmp/fix_sidebar.php
<?php
$path = 'd:/Horntech/resources/views/admin/body/sidebar.blade.php';
$lines = file($path);
$newLines = [];
$skipCount = 0;
foreach ($lines as $i => $line) {
    if ($skipCount > 0) {
        $skipCount--;
        continue;
    }
    // Search for Company::first() around line 20
    if (strpos($line, 'Company::first()') !== false && isset($lines[$i-1]) && strpos($lines[$i-1], '@php') !== false) {
        // We found it. We need to remove the @php before, the line itself, and the @endphp after.
        array_pop($newLines); // remove @php
        $skipCount = 1; // skip @endphp
        continue;
    }
    $newLines[] = $line;
}
file_put_contents($path, implode('', $newLines));
echo "SUCCESS";
?>
